### PR TITLE
Cleanup dead code and fix some imports.

### PR DESCRIPTION
**Pull request by sodul**
This removes code that was here for older versions of python that we no longer support. Unused imports have been removed while missing ones have been added.

### CODE REVIEW

File: green/config.py [hunk 1]
```diff
@@ -441,7 +441,7 @@ def parseArguments(argv=None):  # pragma: no cover
             ),
             default=argparse.SUPPRESS,
         )
-    ),
```
Comment (sodul) [inline green/config.py:444]: This made a tuple declaration serving no purpose.

File: green/test/test_runner.py [hunk 2]
```diff
@@ -413,12 +413,13 @@ def testPasses(self):
...
     def test_run_coverage(self):
         """
-        Running coverage in process mode doesn't crash
+        Running coverage in process mode doesn't crash.
         """
         try:
             import coverage
         except ImportError:
             self.skipTest("Coverage needs to be installed for this test")
+        self.assertTrue(coverage.version_info)
```
Comment (sodul) [inline green/test/test_runner.py:422]: This double up as a way to 'use' coverage so linters do not complained about unused imports.

File: green/result.py [hunk 3]
```diff
@@ -737,14 +735,6 @@ def printErrors(self):
             if not self.args.no_tracebacks:
                 relevant_frames = []
                 for i, frame in enumerate(err.traceback_lines):
-                    # Python2 tracebacks containing unicode need some special handling
```
Comment (IanFoertsch-clumio) [inline green/result.py:740]: I can see the [minimum python version is now 3.8](https://github.com/CleanCut/green/blob/main/setup.cfg#L69), so this make sense 👍

**Review by IanFoertsch-clumio — APPROVED** [review]
👍 I've manually validated all the import-related changes here, and removing the old python 2 related code makes sense as 3.8 is now the minimum version.

**Pull request merged**